jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -646,11 +648,27 @@ public Classpath getClasspath()
         return classpath;
     }
 
+    /**
+     * The list of selected Modules to enable based on configuration.
+     * @return the list of selected modules (by name) that the configuration is requesting.
+     */
     public List<String> getEnabledModules()
     {
         return this.modules;
     }
 
+    /**
+     * Get the Stream of {@link Module} instances that have been enabled successfully.
+     * Filtering out any that are invalid (such as modules that were not, or cannot be enabled).
+     * @return the stream of {@link Module} instances
+     */
+    public Stream<Module> getValidEnabledModulesStream()
```

Comment:
This is a convenience method that makes usage predictable.
This addressed the NPE directly.